src/sentry/static/sentry/app/views/alerts/details/sideHeader.tsx

```diff
@@ -1,40 +0,0 @@
-import React from 'react';
-import isPropValid from '@emotion/is-prop-valid';
-import styled from '@emotion/styled';
-
-import space from 'app/styles/space';
-
-type Props = {
-  className?: string;
-  loading?: boolean;
-  children: React.ReactNode;
-};
-
-const SideHeader = styled(function SideHeader({className, loading, children}: Props) {
-  return (
-    <h6 className={className}>
-      <Title loading={loading}>{children}</Title>
```

Comment:
This has gone through several iterations, but would be nice to have the sidebar always show the Alert Rule + Query titles since they now should always exist, and have placeholder bars underneath it to fit the loading of the rest of the view.

Reply:
Ahhh, totally missed that it was doing this

Reply:
Okay added this back